html/pfappserver/lib/pfappserver/Form/Config/PKI_Provider/packetfence_pki.pm

```diff
@@ -144,13 +144,23 @@ has_field 'organization' => (
 has_field 'ca_cert_path' => (
     type        => 'Path',
     label       => 'CA cert path',
-    required    => 1,
+    required    => 0,
     tags        => {
         after_element   => \&help,
         help            => 'Path of the CA certificate that will generate your certificates',
     },
 );
 
+has_field 'ca_cert_path_upload' => (
+   type => 'PathUpload',
+   accessor => 'ca_cert_path',
+   config_prefix => '.crt',
+   label => 'File Upload',
+   required => 0,
+   upload_namespace => 'pki',
+   element_class => ['input-xxlarge'],
```

Comment:
All these element_class and label don't seem relevant to me since they are not consumed by the frontend nor included in the OPTIONS

Reply:
An artifact from copying and pasting code.
When should I take the time to remove it for all the forms?

Reply:
I don't think we should remove them from all the forms now but at least not add them to the new fields we're adding into the forms